### ISSUE TITLE
Release version v0.6.0

### ISSUE THREAD

**Issue by aloker** [graphql-dotnet/relay#27]
I suggest pulling #26 and releasing v0.6.0. We've been successfully using the changes in production for quite some time now.

**Comment by tvildo** [comment]
Any news on this ?

**Comment by jim-paton** [comment]
Current 0.5.0 release doesn't appear to be compatible with latest 2.4.0 release of GraphQL.Net, I've had to build locally in order to get it working. Any ETA on when an updated release might be likely?

**Comment by fiyazbinhasan** [comment]
Hello contributors! I have one question. The package published on Nuget is very old (published in late 2017) and as mentioned it doesn't work with the latest version of GraphQL.NET. I've been using this library by compiling the repo source and not having any issues. Any chance we can have a 0.6.0 release on NuGet?

**Comment by sungam3r** [comment]
It will be 1.0.0. See https://github.com/graphql-dotnet/relay/milestone/1 . This project is not in the focus of my attention. Of course, it has not been updated for a long time. I just do not have enough time and strength to support all projects that I interested in. @Shane32 may help. Also we need to configure CI pipeline and some other things like code style.

**Comment by Shane32** [comment]
I wouldn't mind releasing it now as 0.6.0, but I don't think the current state of the code is worthy of a 1.0.0 release.

It needs:
- All example and sample projects to run properly (they do not do so now)
- Sample projects' dependencies to be brought up to date
- Basic tests that exercise each sample project
- Basic tests for the core functionality provided
- APIApprovals tests added
- GitHub Action workflows for tests and publishing
- Add code style rules and formatting tests
- Review of included public members to see if they are appropriate and do not mirror existing functionality (e.g. `MutationInputs` is identical to `Inputs`)
- Review of if/how this project would be useful in a typical scenario (with a backing database/repository) versus only for in-memory databases
- Basic review of how these classes can interact with the other portions of the published libraries, such as dataloaders or authentication.
- Review if connection builders are provided and/or how they interact or conflict with the connection builders provided with the main GraphQL.NET library.

These are all tasks that should be completed regardless of the feature set provided.  None of it is very hard, but it does take time.

However, a 0.6.0 release would allow other developers to test and provide more feedback and/or help to this project.

If this sounds good, I can add in some GitHub Action scripts so it will publish, and we can release 0.6.0 once #88 is merged.  I don't have the time to put in the necessary to work on this to bring it to a 1.0.0, and it sounds like @sungam3r doesn't either.  So unless someone else volunteers to work on everything that needs to be done, it may be beneficial for the community to release 0.6.0 in the meantime.  The currently published 0.5.0 release is useless to anyone, as it depends on GraphQL.NET 0.14.0.633.

**Comment by sungam3r** [comment]
Agree. One thing I ask you is to post all the mentioned above tasks as issues.

**Comment by Shane32** [comment]
@fiyazbinhasan I'll release 0.6.0 once #104 and #88 are merged.
